modules/special-rules/pages/war-engines-in-firefights.adoc

```diff
@@ -2,5 +2,6 @@
 
 War engines participate in firefights in much the same way as other units.
 
-* Count the Firepower of each weapon system that can draw a line of fire to at least one enemy unit within 15 cm.
+* Count the Firepower of each weapon system that has at least one enemy unit within fire arc and within 15 cm.
+//IJW editing note: firefights aren't restricted by line of fire, so this has to refer to the weapon arc instead.
```

Comment:
Ah yes, good catch. Just a tweak to correct 'fire arc' to 'weapon arc' to be consistent with our definition.
```suggestion
* Count the Firepower of each weapon system that has at least one enemy unit within its weapon arc and within 15 cm.
```